packages/react-components/src/components/VideoTile.tsx

```diff
@@ -229,6 +261,27 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
   const canShowLabel = showLabel && (displayName || (showMuteIndicator && isMuted));
   const participantStateString = participantStateStringTrampoline(props, locale);
 
+  /* @conditional-compile-remove(pinned-participants) */
+  const optionsMenuItems = (menuItems: VideoTileMenuItems): IContextualMenuItem[] => {
```

Comment:
make sure to memoize this

Reply:
(*this is the actual part that needs memoized: `{ items: optionsMenuItems(menuItems) }`

Reply:
`const optionsMenuItems =` is just a helper function, consider moving to a util file to avoid bloating this file
```tsx
/// VideoTileUtils.ts

export const mapMenuItemsToContextualMenuItems => ...
```